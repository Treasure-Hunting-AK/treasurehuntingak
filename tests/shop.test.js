/* eslint-disable no-undef */
import { render } from '@testing-library/react';
import Shop from '../pages/index';

describe('Index.js', () => {
  it('renders correctly', () => {
    const tree = render(<Shop />);
    expect(tree).toMatchSnapshot();
  });
});
