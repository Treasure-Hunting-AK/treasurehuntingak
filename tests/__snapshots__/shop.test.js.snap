// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index.js renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <main>
          <h1>
            Welcome to Treasure Hunting AK
          </h1>
          <p>
            Discover Alaska
          </p>
          <section>
            <div>
              <h3>
                About
              </h3>
            </div>
            <div>
              <h3>
                Contact
              </h3>
            </div>
            <div>
              <h3>
                Shop
              </h3>
            </div>
          </section>
        </main>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <main>
        <h1>
          Welcome to Treasure Hunting AK
        </h1>
        <p>
          Discover Alaska
        </p>
        <section>
          <div>
            <h3>
              About
            </h3>
          </div>
          <div>
            <h3>
              Contact
            </h3>
          </div>
          <div>
            <h3>
              Shop
            </h3>
          </div>
        </section>
      </main>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
